lxc/init.go

```diff
@@ -2,15 +2,72 @@ package main
 
 import (
 	"fmt"
-	"os"
 	"strings"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.